tests/Behat/Context/GroupsContext.php

```diff
@@ -115,4 +152,127 @@ public function thisAttributesShouldBeVisibleNextToTheGroup(string $group, Table
             $this->indexPage->areAttributesVisible($group, $attributes)
         );
     }
+
+    /**
+     * @Given the store has a product attribute group :groupName with attributes:
+     */
+    public function theStoreHasAProductAttributeGroup(string $groupName, TableNode $table)
+    {
+        $group = new Group();
+        $group->setName($groupName);
+
+        $attributeNames = array_merge([], ...$table->getRows());
+
+        foreach ($attributeNames as $attributeName) {
+            $attributeCode = StringInflector::nameToCode($attributeName);
+
+            $attributeTranslation = new ProductAttributeTranslation();
+            $attributeTranslation->setName($attributeName);
+            $attributeTranslation->setLocale('en_US');
+
+            $syliusAttribute = new ProductAttribute();
```

Comment:
you can use factory

Reply:
referring to our earlier conversation - this will be corrected in opensource